astroquery/solarsystem/pds/core.py

```diff
@@ -0,0 +1,354 @@
+# 1. standard library imports
+
+import re
+import warnings
+
+# 2. third party imports
+from astropy.time import Time
+from astropy import table
+from astropy.io import ascii
+import astropy.units as u
+from astropy.coordinates import EarthLocation, Angle
+from bs4 import BeautifulSoup
+
+# 3. local imports - use relative imports
+# all Query classes should inherit from BaseQuery.
+from ...query import BaseQuery
+from ...utils import async_to_sync
+
+# import configurable items declared in __init__.py, e.g. hardcoded dictionaries
+from . import conf
+
+
+@async_to_sync
+class RingNodeClass(BaseQuery):
+    """
+    a class for querying the Planetary Ring Node ephemeris tools
+    <https://pds-rings.seti.org/tools/>
+    """
+
+    def __init__(self, timeout=None):
+        '''
+        Instantiate Planetary Ring Node query
+        '''
+        super().__init__()
+        self.url = conf.pds_server
+        self.timeout = conf.timeout
+        self.planet_defaults = conf.planet_defaults
```

Comment:
This will prevent changing configuration items at runtime from having any effect. The correct thing to do would be to follow the template in #2341. I have added a section to the opening post there that explains why that is the best solution.